tests/commands/base/test_cookiecutter_logging.py

```diff
@@ -42,3 +45,20 @@ def test_git_stdlib_logging(base_command, logging_level, handler_expected):
 
     # reset handlers since they are persistent
     logging.getLogger("cookiecutter").handlers.clear()
+
+
+def test_cookiecutter_undefined_variable_in_template(base_command):
```

Comment:
This is a good test, but in the wrong location - this test file is for testing the *logging* passthrough of cookiecutter, not the handling of errors generated by cookiecutter (The giveaway is that there aren't any tests of the *other* error modes for generate_template).

It's generally better to keep similar tests together; in this case, `tests/commands/create/test_generate_app_template.py` would be a better home, as that's where the other failure modes for template generation are located.